### PR TITLE
[flink-cdc] rename group.id from 'com.alibaba.ververica' to 'com.ververica'

### PR DESCRIPTION
**Pull request by leonardBang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,18 +56,18 @@ Include following Maven dependency (available through Maven Central):
...
 ```java
 import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
 import org.apache.flink.streaming.api.functions.source.SourceFunction;
-import com.alibaba.ververica.cdc.debezium.StringDebeziumDeserializationSchema;
-import com.alibaba.ververica.cdc.connectors.mysql.MySqlSource;
+import com.ververica.cdc.debezium.StringDebeziumDeserializationSchema;
+import com.ververica.cdc.connectors.mysql.MySqlSource;
 
 public class MySqlBinlogSourceExample {
   public static void main(String[] args) throws Exception {
```
Comment (wuchong) [inline README.md:73]: `SourceFunction<String> sourceFunction = MySQLSource.<String>builder()` should be `MySqlSource`.

File: flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/DebeziumSourceFunction.java [hunk 2]
```diff
@@ -72,9 +73,6 @@
 import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.TimeUnit;
 
-import static com.alibaba.ververica.cdc.debezium.utils.DatabaseHistoryUtil.registerHistory;
-import static com.alibaba.ververica.cdc.debezium.utils.DatabaseHistoryUtil.retrieveHistory;
```
Comment (wuchong) [inline flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/DebeziumSourceFunction.java:76]: It seems many the static import is lost, is that on purpose?

Comment (leonardBang) [reply flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/DebeziumSourceFunction.java:76]: No, I'll update all static imports

**Review by wuchong — COMMENTED** [review]
LGTM.

**Review by wuchong — APPROVED** [review]

**Pull request merged**